README.md

```diff
@@ -0,0 +1,49 @@
+# Pixelpact 📸
+
+Pixeplact gets you started quickly with visual testing for the web.
+
+In contrast to functional or snapshot testing, visual testing verifies that the actually rendered HTML/CSS/JS of your web app truly looks the way it is supposed to. Pixelpact does this by taking screenshots and comparing them to reference images stored in your git repository (using [Git LFS](https://git-lfs.github.com/)).
+
+Checkout our [Getting Started Guide](./getting-started.md).
+
+## Goals
+
+Visual testing is not a new idea and there are plenty of other tools, libraries and service providers that offer a similar functionality as we do.In fact, Pixeplact took a lot of inspiration from existing solutions and is based on their hard work! We created Pixelpact with the following goals:
+
+* **Low entry barrier**. We created Pixeplact to provide a solution to get you started quickly with visual testing, without any vendor lock-in or service provider discussions. Start using Pixelpact today and start growing with visual testing.
+* **Integrated in Git**. Git is the tool that we use on a daily basis and we have a lot of tools built on top of it, for example for code reviews. Because all our test are stored within our git repositories visual tests should live there too.
```

Comment:
- rewording suggestion: `Git is an integral part of our work setting and lot of tools are built on top of it, e.g. code reviews. Since all our tests are stored in git repositories [...]`